src/style.css

```
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf),
+    url(./fonts/Avenir-Heavy.ttf),
+    url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
```

Use separate font-face for each font weight (one for normal, one for bold)
```suggestion
@font-face {
  font-family: "Avenir";
  src: url(./fonts/Avenir-Book.ttf);
  font-weight: 300;
}

@font-face {
  font-family: "Avenir";
  src: url(./fonts/Avenir-Heavy.ttf);
  font-weight: 800;
}
```